mono/mini/mini.h

```diff
@@ -1364,7 +1364,9 @@ typedef struct {
 	guint            r4fp : 1;
 	guint            llvm_only : 1;
 	guint            domainvar_inited : 1;
+#ifdef MONO_ARCH_SIMD_INTRINSICS
```

Comment:
Makes no sense to throw ifdefs for his.

Reply:
Maybe similar to the enum. I can see the point, but I prefer fewer ifdefs, instead of the slight optimization or slightly increased static checking. (while meanwhile I do like static checking, so maybe that is contradictory).